include/envoy/network/connection.h

```diff
@@ -72,11 +72,12 @@ class Connection : public Event::DeferredDeletable, public FilterManager {
 public:
   enum class State { Open, Closing, Closed };
 
-  struct BufferStats {
+  struct ConnectionStats {
     Stats::Counter& read_total_;
     Stats::Gauge& read_current_;
     Stats::Counter& write_total_;
     Stats::Gauge& write_current_;
+    Stats::Counter* bind_errors_;
```

Comment:
Can you add a comment here on why this is a pointer rather than ref (upstream/downstream..)?